Makefile

```diff
@@ -1,22 +1,51 @@
-TOOLS = docker-compose run --rm tools
-CONSOLE = ./bin/console
+BIN=docker-compose
+APP=$(BIN) exec -T app
+CONSOLE=$(APP) bin/console
+TOOLS=docker-compose run --rm tools
 
-all: boot install assets run
+.PHONY: clean
 
-boot:
-	docker-compose up -d
+start: run dependencies db assets permissions
 
-install:
-	$(TOOLS) sh -c "composer install && yarn install"
+clean: cc permissions
 
-assets:
-	$(TOOLS) sh -c "npm run build-dev"
+dependencies: composer yarn permissions
+
+stop:
+	$(BIN) down
 
 run:
-	$(CONSOLE) server:run
```

Comment:
Done. I have to double check but I've handled (in the Makefile) the case where docker is not available on the computer.